libs/device/models/src/lib/import-contacts/unified-contact.ts

```diff
@@ -7,11 +7,37 @@ export type UnifiedContact = {
   id: string
   firstName?: string
   lastName?: string
+  middleName?: string
+  honorificPrefix?: string
+  honorificSuffix?: string
+  nickname?: string
   displayName: string
   phoneNumbers: {
     type?: string
     value: string
     preference?: number
-    label?: string
   }[]
+  emailAddresses: {
+    value: string
+    type?: string
+    preference?: number
+  }[]
+  addresses: {
+    type?: string
+    street?: string
+    streetExtended?: string
```

Comment:
🟢 

Hey, sorry I didn't catch this during our sync. Could we change the field name from streetExtended? to extendedStreet?? I believe this change will bring more consistency to our field naming conventions. What do you think?

```suggestion
    extendedStreet?: string
```